Makefile

```diff
@@ -202,9 +226,27 @@ deploy-azure-resources: set-azure-account # make dev deploy-azure-resources CONF
 validate-azure-resources: set-azure-account # make dev validate-azure-resources
 	az deployment sub create -l "${REGION}" --template-uri "https://raw.githubusercontent.com/DFE-Digital/tra-shared-services/main/azure/resourcedeploy.json" --parameters "resourceGroupName=${RESOURCE_NAME_PREFIX}-${SERVICE_SHORT}-${ENV_SHORT}-rg" 'tags=${RG_TAGS}' "tfStorageAccountName=${RESOURCE_NAME_PREFIX}${SERVICE_SHORT}tfstate${ENV_SHORT}" "tfStorageContainerName=${SERVICE_SHORT}-tfstate" "dbBackupStorageAccountName=${AZURE_BACKUP_STORAGE_ACCOUNT_NAME}" "dbBackupStorageContainerName=${AZURE_BACKUP_STORAGE_CONTAINER_NAME}" "keyVaultNames=['${RESOURCE_NAME_PREFIX}-${SERVICE_SHORT}-${ENV_SHORT}-api-kv', '${RESOURCE_NAME_PREFIX}-${SERVICE_SHORT}-${ENV_SHORT}-inf-kv', '${RESOURCE_NAME_PREFIX}-${SERVICE_SHORT}-${ENV_SHORT}-ui-kv']" --what-if
 
-domain-azure-resources: set-azure-account # make domain deploy-custom-domain CONFIRM_DEPLOY=1
+
+domains-init:  set-azure-pd-subscription ## make [env] domains-init -  terraform init for environment dns/afd resources
+	./bin/terrafile -p terraform/domains/environment_domains/vendor/modules -f terraform/domains/environment_domains/config/${DEPLOY_ENV}_Terrafile
+	terraform -chdir=terraform/domains/environment_domains init -reconfigure -upgrade -backend-config=config/${DEPLOY_ENV}_backend.tfvars
+
+domains-plan: domains-init ## terraform plan for environment dns/afd resources
+	terraform -chdir=terraform/domains/environment_domains plan -var-file config/${DEPLOY_ENV}.tfvars.json
+
+domains-apply: domains-init ## terraform apply for environment dns/afd resources, needs CONFIRM_DEPLOY=1 for production
+	terraform -chdir=terraform/domains/environment_domains apply -var-file config/${DEPLOY_ENV}.tfvars.json
+
+domains-infra-init: bin/terrafile set-azure-account ## make [domain|aks_domain] domains-infra-init -  terraform init for dns/afd core resources, eg Main FrontDoor resource
```

Comment:
This is failing with:
```shell
$ make domains-infra-plan 
echo "Logging on to "
Logging on to 
az account set -s 
argument --subscription/-s/--name/-n: expected one argument
```

Replacing set-azure-account with set-azure-pd-subscription fixes it